src/library/plugin/index.ts

```diff
@@ -50,3 +63,7 @@ export const verifyProof = (
       pluginName
     )
   );
+
+// TODO DEV NOTE:
+// the lib/plugin modules should be required to build a plugin.
+// The stuff that is needed to operate them should be exported from the server library.
```

Comment:
You mean keeping them in a separate package?

Reply:
Just a change of the location of files. lib/plugin -> lib/server